src/pages/Overview/index.js

```diff
@@ -1,16 +1,880 @@
+          </div>
+        ),
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: 'Scheduled for deletion on',
+        dataIndex: 'deletion',
+        key: 'deletion',
+        render: text => {
+          const deleteDate = moment(new Date(Date.parse(text)));
+          const currDate = moment(new Date());
+          const remainingDays = deleteDate.diff(currDate, 'days');
+          if (remainingDays > 60) {
+            return (
+              <div>
+                <Text>
+                  {moment(text)
+                    .add(7, 'days')
+                    .format('YYYY-MM-DDTHH:mm:ss:SSSSSS')}
+                </Text>
+                <Progress
+                  min={0}
+                  max={expirationLimit}
+                  value={expirationLimit - remainingDays}
+                  size={ProgressSize.sm}
+                  measureLocation={ProgressMeasureLocation.none}
+                />
+              </div>
+            );
+          }
+          return (
+            <div>
+              <span>
+                {moment(text)
+                  .add(7, 'days')
+                  .format('YYYY-MM-DDTHH:mm:ss:SSSSSS')}
+              </span>
+              <Progress
+                min={0}
+                max={expirationLimit}
+                value={expirationLimit - remainingDays}
+                size={ProgressSize.sm}
+                variant={ProgressVariant.danger}
+                measureLocation={ProgressMeasureLocation.none}
+              />
+            </div>
+          );
+        },
+      },
+      {
+        title: '',
+        dataIndex: 'fav',
+        key: 'fav',
+        render: () => <Icon type="star" onClick={() => this.onFavourite()} />,
+      },
+      {
+        title: '',
+        dataIndex: 'action',
+        key: 'action',
+        render: (value, row) => {
+          return (
+            <div>
+              <EllipsisVIcon
+                onClick={() => this.showDrowpdown(`newrun${row.key}`)}
+                className="dropbtn"
+              />
+              <div className={styles.dropdown} id={`newrun${row.key}`} style={{ display: 'none' }}>
+                <div className={styles.dropdownContent}>
+                  <div className={styles.dropdownLink} onClick={() => this.deleteResult(row)}>
+                    Delete
+                  </div>
+                </div>
+              </div>
+            </div>
+          );
+        },
+      },
+    ];
+
+    const rowSelection = {
+      onChange: (selectedRowKeys, selectedRows) => {
+        console.log(`selectedRowKeys: ${selectedRowKeys}`, 'selectedRows: ', selectedRows);
+      },
+      getCheckboxProps: record => ({
+        disabled: record.name === 'Disabled User', // Column configuration not to be checked
+        name: record.name,
+      }),
+    };
+
     return (
-      <div>
-        <h1>Overview Page</h1>
+      <div className={styles.paddingBig}>
```

Comment:
Once we've collected more feedback regarding the Overview design, we should consider moving markup in subcomponents in order to improve organization and performance.